web/src/services/theme/reearthTheme/common/metrics.ts

```diff
@@ -1,6 +1,7 @@
 const metrics = {
-  propertyMenuWidth: 308,
+  propertyMenuWidth: 240,
```

Comment:
Why do you thik the property menu should be so narrow? In figma it is 308..

Reply:
I'll do this.